subiquity/cmd/tui.py

```diff
@@ -63,9 +63,9 @@ def parse_options(argv):
     parser.add_argument('--machine-config', metavar='CONFIG',
                         dest='machine_config',
                         help="Don't Probe. Use probe data file")
-    parser.add_argument('--uefi', action='store_true',
-                        dest='uefi',
-                        help='run in uefi support mode')
+    parser.add_argument('--bootloader',
+                        choices=['none', 'bios', 'prep', 'uefi'],
+                        help='Override style of bootloader to use')
```

Comment:
I'm not sure I really understand why this needs to be overridable; surely it's set in stone if you want to boot, that on a particular architecture you need a particular bootloader. No use in having uefi grub on ppc64el, it just won't boot.

Reply:
It's just for testing, yes (this flag is only inspected if --dry-run is also passed)